ppr-ui/src/components/parties/composables/useDebtor.ts

```diff
@@ -115,7 +115,14 @@ export const useDebtor = (props, context) => {
         }
       }
     }
-
+    // format the original address before the compare
+    if (localState.originalDebtor) {
+      localState.originalDebtor.address = formatAddress(localState.originalDebtor.address)
+    }
+    // format the original birthdate the exact same before the compare
+    if (localState.originalDebtor.birthDate && localState.originalDebtor.birthDate.length > 10) {
+      localState.originalDebtor.birthDate = localState.originalDebtor.birthDate.substring(0, 10) + 'T00:00:00-08:00'
```

Comment:
will this register as a change once the time zone changes to 7 hours?

Reply:
just wondering if you'll need to use the date library to add the utc tz

Reply:
changed it to the timezone thingy